doc/homing.md

```diff
@@ -0,0 +1,73 @@
+# Homing {#homing}
```

Comment:
Maybe "Homing / Joint Calibration" so it's cristall clear what we are talking about?